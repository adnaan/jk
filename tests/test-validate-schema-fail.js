import doTest from './validate-schema';

doTest({
  age: -3,
});
