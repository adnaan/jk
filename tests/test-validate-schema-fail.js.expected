Object:
[
  "(root): lastName is required",
  "age: Must be greater than or equal to 0/1"
]
File:
[
  "(root): lastName is required",
  "age: Must be greater than or equal to 0/1"
]
Module:
[
  "(root): lastName is required",
  "age: Must be greater than or equal to 0/1"
]
